src/games/PlayerVsPlayerGame.js

```diff
@@ -1,16 +1,17 @@
-var $ = require('jquery')
+var $ = require('jquery');
 
 function PlayerVsPlayerGame(ui) {
   this.ui = ui;
+  this.playerMarker = this.ui.displayTurn("X");
```

Comment:
Strange behavior setting `this.playerMarker` to the result of this call... Why not just set it to "X" and then pass it in to `this.ui.displayTurn` on the next line?